client/css/style.css

```diff
@@ -568,17 +568,11 @@ kbd {
 	color: #49505a;
 }
 
-#sidebar .chan.active .badge,
-#sidebar .badge:empty {
-	opacity: 0;
-}
```

Comment:
`#sidebar .badge:empty` is already `display: none` (see L562 above), and `#sidebar .chan.active .badge` is a moot point, because [badge content is being emptied on opening](https://github.com/thelounge/lounge/blob/09878a5568e909d0bbcec3a7c6080251424be386/client/js/socket-events/open.js#L10).